search-api/search-actors/src/main/java/org/sunbird/actors/SearchActor.java

```diff
@@ -95,6 +95,9 @@ private SearchDTO getSearchDTO(Request request) throws Exception {
         SearchDTO searchObj = new SearchDTO();
         try {
             Map<String, Object> req = request.getRequest();
+            String secureContent = (String) req.get(SearchConstants.searchSecureContent);
+            if (!StringUtils.isEmpty(secureContent) && secureContent.equals("true"))
+                searchObj.addAdditionalProperty(SearchConstants.searchSecureContent, secureContent);
```

Comment:
We suppose to use the attribute created by Wilky ?? instead of adding new one ?
Refer PR #55